From d5cecd656ea0eb5be54080455cd751150866f66b Mon Sep 17 00:00:00 2001
From: Brian Kim <brian.kim@hardkernel.com>
Date: Wed, 23 Aug 2017 14:09:37 +0900
Subject: [PATCH] ARM: dts: exynos: Add timer node for exynos5420

This patch adds timer node for exynos542x/5800.
Generic Timer Architecture is needed by KVM.

[Brian: fix different lines for odroidxu4-4.9.y branch]
Signed-off-by: Adel Belhouane <bugs.a.b@free.fr>
Signed-off-by: Brian Kim <brian.kim@hardkernel.com>
---
 arch/arm/boot/dts/exynos5420.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/exynos5420.dtsi b/arch/arm/boot/dts/exynos5420.dtsi
index 8b9793256ea53..919bec041194b 100755
--- a/arch/arm/boot/dts/exynos5420.dtsi
+++ b/arch/arm/boot/dts/exynos5420.dtsi
@@ -1321,6 +1321,16 @@
 		       #include "exynos5420-trip-points.dtsi"
 		};
 	};
+
+	timer {
+		compatible = "arm,cortex-a15-timer",
+			     "arm,armv7-timer";
+		interrupts = <1 13 0xf08>,
+			     <1 14 0xf08>,
+			     <1 11 0xf08>,
+			     <1 10 0xf08>;
+		clock-frequency = <24000000>;
+	};
 };
 
 &dp {
